File: new/atoms/tz-score-rules.test.mjs
import * as m from './tz-score-rules.mjs';
const SNAP = {"TZ_SCORE_RULES":"{\"emptyPts\":10,\"ilsPerPoint\":50,\"streakDays\":60,\"streakPts\":5}"};
let f = 0;
for (const [k, s] of Object.entries(SNAP)) if (JSON.stringify(m[k]) !== s) { console.error('✗ ' + k + ' סטה מהצילום'); f = 1; }
if (f) process.exit(1); console.log('✓ tz-score-rules: צילום-ערך תואם — ירוק');

File: new/dart-maor/tz-score-rules_test.dart
// בדיקת-חוזה (רתמת-זהב) · tz-score-rules — צילום-ערך jsonEncode≡JSON.stringify.
// חוללה מכנית (gen-data-dart). הרצה: dart run --enable-asserts ⇒ OK
import 'dart:convert';
import 'tz-score-rules.dart';

void main() {
  // ‏1) צילום-ערך: jsonEncode ≡ JSON.stringify של המקור (זהה-ביט).
  if (jsonEncode(tzScoreRules) != '{"emptyPts":10,"ilsPerPoint":50,"streakDays":60,"streakPts":5}') {
    throw StateError('FAIL tzScoreRules: צילום-הערך סטה');
  }
  print('OK tz-score-rules: 1 data-snapshot(s) passed');
}
